inspirehep/records/marshmallow/literature/common/conference_info_item.py

```diff
@@ -14,22 +14,26 @@
 class ConferenceInfoItemSchemaV1(Schema):
     titles = fields.Raw()
     control_number = fields.Raw()
+    page_start = fields.Raw()
+    page_end = fields.Raw()
+    acronyms = fields.Raw()
 
     @pre_dump
     def resolve_conference_record_as_root(self, pub_info_item):
         conference_record = pub_info_item.get("conference_record")
         if conference_record is None:
             return {}
 
+        _, recid = PidStoreBase.get_pid_from_record_uri(conference_record.get("$ref"))
         try:
-            _, recid = PidStoreBase.get_pid_type_from_endpoint(
-                conference_record.get("$ref")
+            conference = InspireRecord.get_record_by_pid_value(
+                pid_value=recid, pid_type="con"
             )
-        except TypeError:
+        except PIDDoesNotExistError:
             return {}
 
-        conference = InspireRecord.get_record_by_pid_value(recid, "con")
-
         titles = conference.get("titles")
```

Comment:
shouldn't this be `acronyms` now, because we need acronyms to be able display.

Reply:
and need test for this.

Reply:
Actually, if we don't have acronyms, we fall back to the titles so we ultimately need titles to be able to display.